extensions/stac-eo-spec.md

```diff
@@ -132,6 +81,14 @@ The band's common_name is the name that is commonly used to refer to that band's
 |lwir11    | 10.5 - 11.5 |       |           |     10     |          |      31   
 |lwir12    | 11.5 - 12.5 |        |          |     11     |          |      32
 
-### Sample data
 
-See the [examples/](examples/) folder for a full landsat definition.
\ No newline at end of file
+## Using Collections
```

Comment:
I like that this section exists a lot, but I think it'd be good to move it further up in the document, and maybe even group the field additions in to collection vs item level. These would be 'recommendations' - it is good to give the flexibility for data that may want to change some of the values. But I think we should promote the collection concept more in the EO spec, while also getting across that you don't _have_ to use it.